src/test/scala/uk/gov/hmrc/perftests/dds/requests/AuthLoginStubRequests.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.perftests.dds.requests
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.http.request.builder.HttpRequestBuilder
+import uk.gov.hmrc.perftests.dds.Configuration
+
+object AuthLoginStubRequests extends Configuration {
+
+  val signInRoute: String    = s"$authLoginStubUrl/auth-login-stub/gg-sign-in"
+  val digitalDisclosureRoute = s"$digitalDisclosureUrl/digital-disclosure"
+
+  val navigateToSignIn: HttpRequestBuilder =
+    http("Navigate to /auth-login-stub/gg-sign-in")
+      .get(signInRoute)
+      .check(status.is(200))
+
+  def signIn(nino: String): HttpRequestBuilder =
```

Comment:
You only need to define the bare minimum for auth login stub to function properly, there is no need to send all fields as empty strings as part of your request.